problem_builder/templates/html/mcqblock.html

```diff
@@ -1,26 +1,29 @@
-<fieldset class="choices questionnaire">
+<fieldset class="choices questionnaire" aria-describedby="description_{{ self.html_id }}">
```

Comment:
Is the `aria-describedby` here necessary? I am not sure, but I think that presence of the `legend` element that contains the description is enough.